huaweicloud/data_source_huaweicloud_gaussdb_mysql_configuration.go

```diff
@@ -36,7 +36,7 @@ func dataSourceGaussdbMysqlConfigurations() *schema.Resource {
 func dataSourceGaussdbMysqlConfigurationsRead(d *schema.ResourceData, meta interface{}) error {
 	config := meta.(*Config)
 
-	client, err := config.initServiceClient("gaussdb", GetRegion(d, config), "mysql/v3")
+	client, err := config.NewServiceClient("gaussdb", GetRegion(d, config))
```

Comment:
better to wrap up a mysql client in config file like others instead of reference to NewServiceClient directly. Please help to replace it although it's not related to this PR.

Reply:
sure, I will do it latter.